go/cmd/vtctldclient/internal/command/cells.go

```diff
@@ -24,10 +24,55 @@ import (
 
 	"vitess.io/vitess/go/cmd/vtctldclient/cli"
 
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
 )
 
 var (
+	// AddCellInfo makes an AddCellInfo gRPC call to a vtctld.
+	AddCellInfo = &cobra.Command{
+		Use:   "AddCellInfo --root <root> [--server-address <addr>] <cell>",
+		Short: "Registers a local topology service in a new cell by creating the CellInfo.",
+		Long: `Registers a local topology service in a new cell by creating the CellInfo
+with the provided parameters.
+
+The address will be used to connect to the topology service, and Vitess data will
+be stored starting at the provided root.`,
+		DisableFlagsInUseLine: true,
+		Args:                  cobra.ExactArgs(1),
+		RunE:                  commandAddCellInfo,
+	}
+	// AddCellsAlias makes an AddCellsAlias gRPC call to a vtctld.
+	AddCellsAlias = &cobra.Command{
+		Use:   "AddCellsAlias --cells <cell1,cell2,...> [--cells <cell3> ...] <alias>",
```

Comment:
Just curious: why is the `--cells`  flag defined twice?

Reply:
I am still unsure how to best convey this in usage strings, but `pflag.StringSlice` lets you do:

```
-x "a,b" # result: [a, b]
-x "a" -x "b" # result: [a, b]
-x "a,b" -x "c" # result: [a, b, c]
```